<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrectionTab</class>
 <widget class="QWidget" name="TextureTab">
    <property name="styleSheet">
   <string notr="true">QGroupBox {
border: 1px solid grey;border-radius: 10px;margin-top: 1ex; margin-right: 0ex
}
QGroupBox:title {
                           subcontrol-origin: margin;
                           padding: 0 3px;
                           subcontrol-position: top center;
                           padding-top: 0px;
                          padding-bottom: 0px;
                           padding-right: 10px;
                            color: rgb(56, 56, 56)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- Load Data Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_loadData">
     <property name="title">
      <string>Load Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_load">
      <item>
       <widget class="FileFinderWidget" name="finder_texture_ws" native="true"/>
      </item>
        <item>
   <widget class="QPushButton" name="btn_loadWSFiles">
    <property name="text">
     <string>Load Workspace Files</string>
    </property>
   </widget>
  </item>
         <item>
       <widget class="FileFinderWidget" name="finder_texture_tables" native="true"/>
      </item>
        <item>
   <widget class="QPushButton" name="btn_loadParamFiles">
    <property name="text">
     <string>Load Parameter Files</string>
    </property>
   </widget>
  </item>
      <item>
       <widget class="QTableWidget" name="table_loaded_data"/>
      </item>
      <item>
 <layout class="QHBoxLayout" name="horizontalLayout_tableButtons">
  <item>
   <spacer name="spacer_left">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
   </spacer>
  </item>
    <item>
   <widget class="QPushButton" name="btn_deleteSelected">
    <property name="text">
     <string>Delete Selected Files</string>
    </property>
   </widget>
  </item>
  <item>
   <widget class="QPushButton" name="btn_selectAll">
    <property name="text">
     <string>Select All Files</string>
    </property>
   </widget>
  </item>
  <item>
   <spacer name="spacer_right">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
   </spacer>
  </item>
 </layout>
</item>
     </layout>
    </widget>
   </item>
 <!-- Scattering Power Section -->
 <item>
 <widget class="QGroupBox" name="groupBox_sample">
  <property name="title">
   <string>Scattering Power Corrections</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_sample">
      <item>
       <widget class="QCheckBox" name="check_absorption">
        <property name="text">
         <string>Include Scattering Power Correction</string>
        </property>
       </widget>
      </item>
   <!-- Crystal Buttons Row -->
   <item>
    <layout class="QHBoxLayout" name="transferSampleLayout">
     <item>
    <widget class="QComboBox" name="combo_workspaceList"/>
   </item>
        <item>
    <widget class="QPushButton" name="btn_setCrystal">
     <property name="text">
      <string>Set Crystal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_copyCrystalToAll">
     <property name="text">
      <string>Copy Crystal to All</string>
     </property>
    </widget>
   </item>
        <item>
 <widget class="QLabel" name="label_10">
  <property name="text">
   <string>HKL: </string>
  </property>
 </widget>
</item>
<item>
 <widget class="QLineEdit" name="hkl_lineedit"/>
</item>

    </layout>
   </item>
  </layout>
 </widget>
</item>

   <!-- Absorption Correction Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_absorption">
     <property name="title">
      <string>Pole Figure Calculation</string>
     </property>
     <layout class="QVBoxLayout">
         <item>
          <widget class="FittingPlotView" name="widget_plot" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
      <widget class="QPushButton" name="btn_calc_pf">
       <property name="text">
        <string>Calculate Pole Figure</string>
       </property>
      </widget>
     </item>
     </layout>
    </widget>
   </item>

  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileFinderWidget</class>
   <extends>QWidget</extends>
   <header>mantidqt.widgets.filefinderwidget</header>
   <container>1</container>
  </customwidget>
     <customwidget>
   <class>FittingPlotView</class>
   <extends>QWidget</extends>
   <header>mantidqtinterfaces.Engineering.gui.engineering_diffraction.tabs.fitting.plotting.plot_view</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
